packages/web-runtime/src/store/modal.ts

```diff
@@ -46,6 +46,10 @@ const actions = {
 
   toggleModalConfirmButton({ commit }) {
     commit('TOGGLE_MODAL_CONFIRM_BUTTON')
+  },
+
+  setModalConfirmButton({ commit }, status) {
```

Comment:
setModalConfirmButton should be setModalConfirmButtonDisabled

Reply:
yup